### PR TITLE
Add Symbols for JCasc Support

### LINKED ISSUE #2: Add a Symbol for the JCasC support
The Clobal Configuration definition does not have a `@Symbol`, so that the YAML entry in JCasC is `globalPluginConfiguration`. It would be nice to add a symbol so that there is a better entru

https://github.com/jenkinsci/tekton-client-plugin/blob/master/src/main/java/org/waveywaves/jenkins/plugins/tekton/client/GlobalPluginConfiguration.java#L15-L16

### PR DESCRIPTION
**Pull request by waveywaves**
fixes #2

### CODE REVIEW

**Comment by waveywaves** [comment]
cc @oleg-nenashev

File: src/main/java/org/waveywaves/jenkins/plugins/tekton/client/GlobalPluginConfiguration.java [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
 import jenkins.model.GlobalConfiguration;
+import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.DataBoundConstructor;
 
 import java.util.logging.Logger;
 
 import static java.util.logging.Level.SEVERE;
 
 @Extension
+@Symbol("Global")
```
Comment (oleg-nenashev) [inline src/main/java/org/waveywaves/jenkins/plugins/tekton/client/GlobalPluginConfiguration.java:14]: Please keep a plugin-specific symbol, e.g. "tekton" or "tektonClient".

File: src/main/java/org/waveywaves/jenkins/plugins/tekton/client/build/create/CreateStep.java [hunk 2]
```diff
@@ -32,11 +34,12 @@
 import org.waveywaves.jenkins.plugins.tekton.client.logwatch.PipelineRunLogWatch;
 import org.waveywaves.jenkins.plugins.tekton.client.logwatch.TaskRunLogWatch;
 
+@Symbol("createStep")
```
Comment (oleg-nenashev) [inline src/main/java/org/waveywaves/jenkins/plugins/tekton/client/build/create/CreateStep.java:37]: `tektonCreateStep` ?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Please keep symbols specific to the Tekton Plugin

Also, when you have `DataboundSetter`, no need to have the same parameters in the `DataboundConstructor`.

**Pull request merged**